microservices/MS1/controllers/notifications/notification-dao.js

```diff
@@ -0,0 +1,13 @@
+import notificationModel from "./notification-model.js";
+
+export const createNotification = (userId, message, type,redirectionUrl=null ) => {
+    return notificationModel.create({ userId, message, type, redirectionUrl });
+    }
+
+export const getNotifications = (userId) => {
+    return notificationModel.find({ userId: userId });
```

Comment:
nit: findOne?

Reply:
no it will be returning all the user notifications user has received, not just one.

Reply:
Makes sense, missed that.